styles/_blog.less

```diff
@@ -70,7 +70,7 @@ body:not(.button-style-default) .sqs-editable-button, body:not(.button-style-def
 }
 
 .blog-author-name {
-  color: @badgerRed;
+  color: @badgerRedOnWhite;
```

Comment:
I think this should be 'border-bottom: 2px solid badgerRedOnWhite;', which is the case in Website Honestly